### PR TITLE
feat: variables refactor

### PR DESCRIPTION
**Pull request by exu**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by vsukhin — APPROVED** [review]

**Review by lreimer — CHANGES_REQUESTED** [review]
I think the changes have been made on an outdated source version. Also should we use a beta001 version as dependency, or better a more stable one?

File: pkg/runner/runner.go [hunk 1]
```diff
@@ -42,9 +42,8 @@ func (r *K6Runner) Run(execution testkube.Execution) (result testkube.ExecutionR
 	args := []string{"run"}
 
 	// convert executor env variables to k6 env variables
-	for key, value := range execution.Envs {
-		env_var := fmt.Sprintf("%s=%s", key, value)
-		args = append(args, "-e", env_var)
+	for _, v := range execution.Variables {
```
[the remark this answers is not in the document]
Comment (exu) [reply pkg/runner/runner.go:45]: 🤦 old main branch - closing this one - will do on top of main branch once more

File: go.mod [hunk 2]
```diff
@@ -3,7 +3,7 @@ module github.com/kubeshop/testkube-executor-k6
 go 1.17
 
 require (
-	github.com/kubeshop/testkube v0.11.14
+	github.com/kubeshop/testkube v1.1.10-beta001
```
[the remark this answers is not in the document]
Comment (exu) [reply go.mod:6]: no with those changes - I'll change those versions after merge testkube to main branch - just want to release everything in one shot

[the remark this answers is not in the document]
Comment (exu) [reply go.mod:6]: It'll be 1.1.10+ after merge

**Pull request closed without merging**